src/tutorials/narratives-how-to-write.rst

```diff
@@ -0,0 +1,349 @@
+Specifically the hostname is *not* used [#f2]_ -- only the pathname is used, and the way the dataset pathname is interpreted is different: Auspice can only access datasets on your computer (in the directory you supply via ``--datasetDir``) whereas nextstrain.org accesses data from :doc:`a range of places </guides/share/index>`.
+
+The following example of a narrative slide may make this difference clearer:
+
+.. code-block:: markdown
+
+  # [Which dataset is this?](https://nextstrain.org/community/inrb-drc/ebola-nord-kivu)
+
+  The dataset displayed here differs if you run it on localhost (auspice)
+  or through nextstrain.org.
+
+  When viewed on nextstrain.org this dataset is sourced from
+  [this GitHub repository](https://github.com/inrb-drc/ebola-nord-kivu)
+  as it is using our
+  [community sharing functionality](https://docs.nextstrain.org/en/latest/guides/share/community-builds.html).
+
+  To work when running on auspice locally, we would need to have a dataset
+  with the following filename: `community_inrb-drc_ebola-nord-kivu.json`.
+
+
+
+This complexity encouraged us to build the interactive debugger we have used in this tutorial and we encourage to use that approach rather than developing your narratives using Auspice locally & then trying to share them through nextstrain.org!
+
+
+Where can I find some example narratives to use as inspiration?
+---------------------------------------------------------------
+
+All of the narratives written by the core Nextstrain team are available in `this GitHub repo <https://github.com/nextstrain/narratives>`__.
+
+Future directions
+-----------------
+
+The app used to test narratives is in its infancy.
+We hope to incrementally add features such as Markdown editing, changing the dataset view settings, and the ability to publish the narrative to :doc:`Nextstrain Groups </learn/groups/index>`.
+The eventual aim is to be able to write & publish an entire narrative from within the app, without needing to know any specifics of the Markdown language behind it.
+
+
+Bugs? Improvements? Suggestions?
+================================
+
+The debugger (as of November 2022) is in a beta-release phase. Please get in touch if you have suggestions or find bugs!
+You can submit an `issue on GitHub <https://github.com/nextstrain/nextstrain.org/>`__ or make a post on our `discussion forum <https://discussion.nextstrain.org/>`__.
+
+
+
+
+--------------
+
+.. rubric:: Footnotes
+
+.. [#f1] The provided string is actually rendered as ``### <abstract>``, so the first line will appear as a h3 heading!
+  Multiple line strings are possible in YAML and we suggest using these.
+
+.. [#f2] In practice, the URL path component is what is used to access the dataset relative to the current hostname, however for historical reasons the protocol and hostname must be either ``http://localhost:4000`` or ``https://nextstrain.org``. 
+
```

Comment:
I think it may have been removed - I noticed this reviewing a separate PR which changes the narrative parsing. I think the current implementation is that the hostname is completely unused?

Reply:
#146 contains 5feeb49bf982f6807c4a386138980a594c220ed2 which removes the footnote.